app/controllers/articles_controller.rb

```diff
@@ -28,8 +29,9 @@ def edit
   end
 
   def create
-    @article = Article.new(article_params)
-
+    @article = Article.new
+    @article.title = params[:article][:title]
```

Comment:
When making the POST request, it would add params such as token and utf. I also tried adding those to params.permit(), but this seemed better.